app/models/concerns/search.rb

```diff
@@ -40,6 +26,17 @@ def status(param)
       param == 'active' ? where(active: true) : where(active: false)
     end
 
+    def keyword(k)
+      tsquery = <<-SQL
+        (to_tsquery('english', ''' ' || '#{k}' || ' ''' || ':*'))
```

Comment:
I was just re-reading the Rails Security Guide a couple days ago and it has some good examples: http://guides.rubyonrails.org/security.html#sql-injection

It would take me a while to figure one out, but think you'd basically want to close the quoted string and the function call to `to_tsquery`, add in whatever other fields are needed plus a new `FROM` clause, then use `--` to comment out the rest of the generated query.

The generated SQL is still going to be a `SELECT` clause, so an attacker probably couldn't destroy the DB with a `DELETE`, but I think it would be possible to construct a query that exposes the `admins` or `users` table.